cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-hcp.adoc

```diff
@@ -31,11 +31,115 @@ In this tutorial, we will create these resources first. We will also set up some
 rosa list regions --hosted-cp
 ----
     
-. Create the VPC. For this tutorial, the following script will create the VPC and its required components for you. It will use the region configured for the `aws` CLI.
+. Create the VPC. For this tutorial, the following link:https://github.com/openshift-cs/rosaworkshop/blob/master/rosa-workshop/rosa/resources/setup-vpc.sh[script] will create the VPC and its required components for you. It will use the region configured for the `aws` CLI.
```

Comment:
One small note here to avoid future tense - "the following script creates..." and "It uses the region..." are preferred in our style guides.

Reply:
Thank you! The future tense is sprinkled throughout the workshop material. I would like to follow up at some point and scrub all of it for consistency